datadog_checks_dev/setup.py

```diff
@@ -72,8 +72,7 @@
             'appdirs>=1.4.4',
             'atomicwrites',
             'beautifulsoup4>=4.9.3',
-            # click 8.0.0 breaks with ddev --version
-            'click==7.1.2',
+            'click~=8.0; python_version > "3.0"',
```

Comment:
What about py2 ?

Reply:
CLI is Python 3-only

Reply:
py2 tests skip cli parts

Reply:
So i guess we don't need the `python_version > "3.0` stuff at all then

Reply:
true, pushed a commit 👍